e2e/cypress/integration/channel/message_permalink_spec.js

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Message permalink', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M13675-Copy a permalink and paste into another channel', () => {
+        const message = 'Hello' + Date.now();
+        const channelName = 'test-message-channel-1';
+
+        // # Create new DM channel with user's email
+        cy.apiGetUsers(['user-1', 'sysadmin']).then((userResponse) => {
+            const userEmailArray = [userResponse.body[1].id, userResponse.body[0].id];
+
+            cy.apiCreateDirectChannel(userEmailArray).then((response) => {
+                const directChannel = response.body;
+                cy.get(`#sidebarItem_${directChannel.name}`).click({force: true});
```

Comment:
Not sure if you run the test using the fresh test data.  I saw this line failed when the DM created was not in the channel sidebar (since nothing to click on it). One way to fix is to visit the URL directly.
```suggestion
                cy.visit('/ad-1/messages/@sysadmin');
```